### PR TITLE
Update firewall.rst

### PR DESCRIPTION
**Pull request by cotosso**
better explanation of ALG

### CODE REVIEW

**Review by filippocarletti — APPROVED** [review]

File: administrator-manual/en/firewall.rst [hunk 1]
```diff
@@ -105,8 +105,13 @@ If hair-pinning is still required, check the :guilabel:`Enable hairpin NAT` opti
 Application Level Gateway (ALG)
 -------------------------------
 
-ALG allows SIP and H.323 protocols to operate through NAT. When enabled, ALG will inspect and rewrite Voice over Internet Protocol (VoIP) network packets and open required ports. 
-ALG is enabled by default, but if you're if you're experiencing audio and call problems with your PBX or VoIP client try to disable it.
+Application-level gateway is a security component that augments a firewall or NAT employed in a computer network.
```
Comment (gsanchietti) [inline administrator-manual/en/firewall.rst:108]: Use the same form found inside the title:
```suggestion
Application level gateway is a security component that augments a firewall or NAT employed in a computer network.
```

File: administrator-manual/en/firewall.rst [hunk 2]
```diff
@@ -105,8 +105,13 @@ If hair-pinning is still required, check the :guilabel:`Enable hairpin NAT` opti
...
+ALG is enabled by default on NethSecurity, it supports many application protocols (like FTP, SIP and so on), allowing them to operate through NAT.
+ALG inspect and rewrite specific network packets and automatically opens required ports.
```
Comment (gsanchietti) [inline administrator-manual/en/firewall.rst:110]:
```suggestion
ALG inspects and rewrites specific network packets and automatically opens required ports.
```

File: administrator-manual/en/firewall.rst [hunk 3]
```diff
@@ -105,8 +105,13 @@ If hair-pinning is still required, check the :guilabel:`Enable hairpin NAT` opti
...
+
+**Enable SIP-ALG**
+
+Since some PBXs may not work properly in the presence of ALG on SIP and H.323 protocols, if you're experiencing audio and call problems with your PBX or VoIP client try to disable SIP-ALG and H.323-ALG.
```
Comment (gsanchietti) [inline administrator-manual/en/firewall.rst:114]: > in the presence 

It's not a common Ensligh form, please rephrase it.

File: administrator-manual/en/firewall.rst [hunk 4]
```diff
@@ -105,8 +105,13 @@ If hair-pinning is still required, check the :guilabel:`Enable hairpin NAT` opti
 Application Level Gateway (ALG)
 -------------------------------
 
-ALG allows SIP and H.323 protocols to operate through NAT. When enabled, ALG will inspect and rewrite Voice over Internet Protocol (VoIP) network packets and open required ports. 
-ALG is enabled by default, but if you're if you're experiencing audio and call problems with your PBX or VoIP client try to disable it.
+Application-level gateway is a security component that augments a firewall or NAT employed in a computer network.
+ALG is enabled by default on NethSecurity, it supports many application protocols (like FTP, SIP and so on), allowing them to operate through NAT.
```
Comment (gsanchietti) [inline administrator-manual/en/firewall.rst:109]: Do not use the product name, but use the `product` macro.

**Review by gsanchietti — CHANGES_REQUESTED** [review]

**Comment by gsanchietti** [comment]
There are still required changes, please request the review again when done.

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**